charts/shoot-addons-kyma/charts/kyma/values.yaml

```diff
@@ -1,12 +1,19 @@
 ---
 jobs:
-  image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20190325-ff66a3a
+  image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20200121-4f3202bd
   delay: 30
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 50m
+      memory: 64Mi
 kyma:
-  version: "1.8.0"
+  version: "1.10.0"
```

Comment:
This is used as a job name above. This won't work. Kubernetes object names must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character.